winlogbeat/eventlog/wineventlog.go

```diff
@@ -62,8 +62,7 @@ var (
 
 const (
 	// renderBufferSize is the size in bytes of the buffer used to render events.
-	renderBufferSize = 1 << 14
-
+	renderBufferSize = 1 << 19 // 512KB, 256K wide characters
```

Comment:
I know you didn't write the original code, but consider `*Bytes` / `*Chars` suffixes so subsequent readers don't have to remember this comment.

Reply:
it's tricky as we're using it for various APIs along the path, `EvtFormatMessage` wants the size in wchars, `EvtRender` in bytes, and anyway our scratch buffer is in bytes.

Reply:
That `1<<19` is always a byte count, regardless of the API being used, right?  Since this `wineventlog` variable is going to be used with wide char strings, we can anticipate some risk of ambiguity.  By naming the variable in a way that clearly states it's a byte (vs character) count, we avoid subsequent ambiguity about whether need to multiply it by 2, divide it by 2, or leave it alone, especially when it's intertwined with other variables that use character counts.

Here we have `renderBufferSize` that's a byte count:  https://github.com/elastic/beats/blob/9bbf354b6fe52e68942c6173209ad2dd82da2018/winlogbeat/eventlog/wineventlog.go#L65

Here we have `bufferSize` that's a character count:  https://github.com/elastic/beats/blob/9bbf354b6fe52e68942c6173209ad2dd82da2018/winlogbeat/sys/wineventlog/wineventlog_windows.go#L413

`renderBufferSize` and `bufferSize` are different units.  That's pretty confusing, right?  What if the first were named `renderBufferSizeBytes` and the latter were named `bufferSizeChars`?

For example, in many places Windows [uses the `cch` prefix](https://learn.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-pathunexpandenvstringsw) for character counts, and [`cb` for byte counts](https://learn.microsoft.com/en-us/windows/win32/api/winreg/nf-winreg-regqueryvalueexw).  I'm not saying we should use that schema, but you get the idea.

Reply:
agree, I think we can assume by default that it's bytes, but this particular function can be improved to provide explicit local aliases

Reply:
refactored 15727de7e911587a2c611b3f861db202c7f22e1a